kuma/wiki/events.py

```diff
@@ -70,7 +70,7 @@ def _mails(self, users_and_watches):
         log.debug('Sending edited notification email for document (id=%s)' %
                   document.id)
         subject = ugettext(
-            u'[MDN] Page "{document_title}" changed by {creator}')
+            u'[MDN] Page "%(document_title)s" changed by %(creator)s')
```

Comment:
That won't be enough since the `emails_with_users_and_watches` function uses new style string formatting locally. We need to update it to use `%` instead.